### ISSUE TITLE
Feature request: show influx in the sidebar

### ISSUE THREAD

**Issue by alranel** [jensmtg/influx#55]
Hello :) 
I'm a happy user of this plugin, which I enabled by default on all my Obsidian notes (although I had to recently disable it because of #40 but that's another story).

I was wondering: would it make sense to add an option to show this plugin in a sidebar instead of appending it to the note body? On one side there might be minor UI constraints to solve given the narrower width, but I think it would make much sense and maybe could also provide a workaround for the aforementioned #40 issue :)

What do you think?

**Comment by Josh2K** [comment]
I agree, it would be very helpful to have influx available on the sidebar.  Thats where i generally view backlinks for my notes as its often nice to be able to view the links side by side with the note.
